jobs/buildhub/utils.py

```diff
@@ -177,36 +198,27 @@ def is_build_url(product, url):
     ):
         return False
 
-    re_exclude = re.compile(
-        '.+(tinderbox|try-builds|partner-repacks|latest|contrib|/0\.|'
-        'experimental|namoroka|debug|sha1-installers|candidates/archived|'
-        'stylo-bindings|/1.0rc/|/releases/win../|dominspector|/test/|testing|'
-        '%28.+%29|\sInstaller\.(\w{2,3}\-?\w{0,3})\.exe)'
-    )
-    if re_exclude.match(url):
+    if _build_url_exclude_names_regex.match(url):
         return False
 
+    extensions = FILE_EXTENSIONS
     # Only .exe for Windows.
-    extensions = list(FILE_EXTENSIONS)
     if 'win' in url:
-        extensions.remove('zip')
+        extensions = [x for x in extensions if x != 'zip']
```

Comment:
Before, it would create a new list object (by cloning the tuple) every single time. Even though it sometimes didn't need to remove `'zip'`.